src/main/java/uk/gov/hmcts/juror/api/moj/utils/NumberUtils.java

```diff
@@ -19,4 +19,8 @@ public static int unboxIntegerValues(Integer integer) {
             :
                 integer;
     }
+
+    public static int calculatePercentage(long numerator, long denominator) {
+        return denominator == 0 ? 0 : (int) Math.round((numerator * 100.0) / denominator);
```

Comment:
The percentage calculation should be 

  public double calculatePercentage(double actual, double total) {
        return total == 0 ? 0 : actual * 100 / total;
    }

Reply:
updated